src/backend/results/new_way/get_results.php

```
@@ -99,7 +98,7 @@ function getinprocess_n($missing, $code)
     return $titles;
 }
 
-function get_results_new($cat, $camp, $depth, $code, $filter_sparql): array
+function get_results_new($cat, $camp, $depth, $code, $filter_sparql, $cat2): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$camp'. (undefined)

(UnusedFormalParameter)